webpack.config.js

```diff
@@ -0,0 +1,83 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const PATHS = {
+  app: path.join(__dirname, 'client/src'),
+  build: path.join(__dirname, 'client/src/build'),
```

Comment:
Why are you creating your builds in your src directory? What's the use of the dist directory? Also I see you have two materialize folders on this project can it be consolidated into one? It's also of no use to version your builds as they change very frequently across different environment.

Reply:
You might also want to consider setting ignore config in babelrc so it does bother about your materialize directory